frontend/src/App.tsx

```diff
@@ -93,36 +99,6 @@ const App = (): React.ReactElement => {
                   path={Routes.EMAIL_ACTION}
                   component={EmailActionHandler}
                 />
-                <PrivateRoute
-                  exact
-                  path={Routes.STUDENT_PAGE}
```

Comment:
do we still want `STUDENT_LANDING`

Reply:
maybe we can add when we create the associated page? i think `StudentPage` is empty rn